src/components/ha-sidebar.js

```diff
@@ -175,6 +175,10 @@ class HaSidebar extends
     return hass.config.core.components.indexOf('mqtt') !== -1;
   }
 
+  computeUserName(user) {
+    return user && (user.name || 'Unnamed User');
```

Comment:
`_computeUserName(user)`

`'Unnamed user'`

also remove package-lock.json from this PR

Reply:
"Unnamed User" used in the user list page